### PR TITLE
DEC-000: Use NVM on GH Action.

### PR DESCRIPTION
**Pull request by sherakama**
# READY FOR REVIEW

### CODE REVIEW

File: .github/workflows/autorelease.yml [hunk 1]
```diff
@@ -39,6 +39,11 @@ jobs:
           ref: ${{ github.base_ref }}
           fetch-depth: 0
 
+      - name: Setup node
+        uses: actions/setup-node@v2
+        with:
+          node-version-file: '.nvmrc'
+
```
Comment (sherakama) [inline .github/workflows/autorelease.yml:46]: From: 
https://github.com/actions/setup-node/issues/32

**Comment by sherakama** [comment]
This might help 🤷  but we also might have to update our dependencies.

File: .github/workflows/autorelease.yml (hunk 1 above)
Comment (jenbreese) [reply .github/workflows/autorelease.yml:46]: It looks right. I'm not sure how I could test it locally.

**Review by jenbreese — APPROVED** [review]
gtg

**Pull request merged**